content/architecture/index.textile

```diff
@@ -0,0 +1,111 @@
+---
+title: Architecture overview
+meta_description: "Learn more about Ably's platform architecture."
+meta_keywords: "architecture"
+---
+
+Ably's platform architecture is built to deliver dependable realtime experiences at global scale. As the definitive realtime experience platform, we serve billions of devices monthly with unmatched reliability and performance.
```

Comment:
> unmatched reliability and performance.

How do we quantify this?

Reply:
I've updated this to focus on scale, and address this under the four pillars